MANIFEST.in

```diff
@@ -1,5 +1,9 @@
-include LICENSE
-include CHANGES.txt
-include README.rst
-graft aiomysql
-global-exclude *.pyc *.swp
+exclude .coveragerc
+exclude .flake8
+exclude .gitignore
+exclude docker-compose.yml
+exclude requirements-dev.txt
+exclude .github/**
+exclude docs/**
+exclude examples/**
+exclude tests/**
```

Comment:
Tests and everything related should be included in sdist because it's used as a source for downstream packaging normally. Besides, it doesn't really make much sense to maintain this exclusion list — just include everything from Git and forget about it. Drop this file altogether.

Reply:
makes sense, removed